packages/flutter/test/material/data_table_test.dart

```diff
@@ -94,9 +106,40 @@ void main() {
     ));
     await tester.pumpAndSettle(const Duration(milliseconds: 200));
 
-    await tester.tap(find.text('375'));
+    await tester.tap(find.text('375'),pointer: 2);
```

Comment:
Spacing nit, here and elsewhere below
```suggestion
    await tester.tap(find.text('375'), pointer: 2);
```

Reply:
The changes on the test file are still pending as well